desk.css

```diff
@@ -0,0 +1,799 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap');
+
+@media only screen and (min-width: 768px) {
```

Comment:
- [ ] Your page has some overflow and cuts some text, please fix that. it will make your site more appealing and nice

![Screenshot_20220427_154836](https://user-images.githubusercontent.com/25227022/165559323-730fe3b4-ffe9-495c-9735-195a7261af62.png) ![Screenshot_20220427_155057](https://user-images.githubusercontent.com/25227022/165559558-90d7a696-c750-4afc-9531-aea267d23edc.png)

Reply:
Hello @Ismailco,
Please at what height and width did you make these screenshots. Am having issues fixing the width.

Reply:
@Christabelle143 I take the screenshots when using the responsive view on chrome (475x718)

Reply:
Hello @Ismailco,
Please can I get the width and height at which you took these screenshots because my pages are displaying well on my web browser and they are responsive as well at different screen sizes . So I would like to get that width and see how it's displaying to better know how to make the changes.

Reply:
And trying to centralize the form at a different width makes the form not centralized when the website on the desktop version

Reply:
@Christabelle143,
#### You can use % or (vh, vw) instead of px or em. this will make your page responsive and use dynamic sizing

Reply:
Thanks